### ISSUE TITLE
Show a warning if a link's href doesn't contain https

### ISSUE THREAD

**Issue by jackdomleo7** [jackdomleo7/Checka11y.css#56]
### Describe the new a11y feature or project enhancement

Linking to a source that is not `https` could be seen as _not good_. While this is not strictly an a11y feature, I feel it is still an important thing to highlight. We should be avoiding using links that aren't `https`.

### Describe the solution you'd like

Show a warning if a link's `href` doesn't contain `https`.

```css
a:not([href*="https"]) {
  ...
}
```

**Comment by IamMattL** [comment]
Im not sure about this, whether its http/https its controlled by the browser, http is fine and if the site your going to supports https it should redirect you to it (given its setup properly) but if you use https and it isnt supported then the link would be broken.

**Comment by SavvasStephanides** [comment]
Not sure about this either. Sure `https` is more secure than `http` but I don't think it should be included as it's not strictly an accessibility feature. Plus it could be a pain when testing on local links.

**Comment by jackdomleo7** [comment]
Ok cool! Thank you @IamMattL @SavvasStephanides. Just a thought I had, but it's probably not needed. 🙂
